tier4/AutowareArchitectureProposal.iv
# Ros2 v0.8.0 autoware planning rviz plugin

# How to check

1. Open rviz and subscribe topic from display type: Max Velocity

2. publish topic and confirm rviz
3. 
```
ros2 topic pub /planning/scenario_planning/current_max_velocity autoware_planning_msgs/msg/VelocityLimit '{data: 10.0}'
```
